controls/roles/update-stereum/tasks/main.yml

```diff
@@ -0,0 +1,32 @@
+---
+- name: Download update metadata
+  uri:
+    url: https://stereum.net/downloads/updates.json
+    method: GET
+    return_content: true
+    status_code: 200
+    body_format: json
+  register: update_data_download
+
+- name: Set update data
+  set_fact:
+    update_data: "{{ update_data_test | default(update_data_download) }}"
+
+- name: Fetch repository information
+  shell: git fetch --all
+  args:
+    chdir: "{{ stereum.settings.controls_install_path }}"
```

Comment:
`stereum.settings.controls_install_path` is by default `/opt/stereum`

`/ansible` is added in Javascript Backend when the repository is pulled.
Therefore does `stereum.settings.controls_install_path` never include `/ansible`

So i would suggest an extra variable or manipulating the path

Reply:
nice catch! done